api/v1/helpers.go

```diff
@@ -69,6 +69,8 @@ const (
 	DefaultServiceClientPort = 5433
 )
 
+var EmptyPassword = ""
```

Comment:
Make it a const

Reply:
I just removed it and used the variable in separate files.